src/App.jsx

```diff
@@ -1,45 +1,96 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { Routes, Route, Navigate, useNavigate, useLocation } from 'react-router-dom';
 import Header from './components/Header/Header';
 import Courses from './components/Courses/Courses';
 import CreateCourse from './components/CreateCourse/CreateCourse';
+import Registration from './components/Registration/Registration';
+import Login from './components/Login/Login';
 import './App.css';
-import { mockedCoursesList, mockedAuthorsList } from './constants';
+import { api_endpoints, storageKeys, paths } from './constants';
```

Comment:
if `api_endpoints` is a constant that's imported from another module, usually its better to format them as uppercased, i.e. `API_ENDPOINTS`. Similarly, you can do `STORAGE_KEYS` and `PATHS`